src/java/org/apache/cassandra/repair/consistent/LocalSessions.java

```diff
@@ -277,7 +277,7 @@ else if (shouldDelete(session, now))
                 {
                     if (!sessionHasData(session))
                     {
-                        logger.debug("Auto deleting repair session {}", session);
+                        logger.info("Auto deleting repair session {}", session);
```

Comment:
The loggings in this class should probably remain at debug level IMO.
These messages are really useful to debug things that could go wrong in how incremental repair is implemented.
While I support getting back memtable flushes and compaction loggings in system.log, these lines will hardly be useful to most operators (but I understand they are to Cassandra devs).
wdyt?

Reply:
If we were seeing this a lot, I'd agree with you, but the amount of logging that will be added to INFO is relatively small, compared with the benefit of understanding what's happening if something goes wrong.  The case here is to understand the problem after the fact.  Since incremental repair is essentially a full rewrite in 4.0, I'm leaning on the side of "a few extra log messages can prove to be very helpful" rather than "maybe people won't need it.".

Reply:
ok, works for me then.